app/api/services/suppliers.py

```diff
@@ -373,3 +382,56 @@ def get_approved_suppliers(self):
         )
 
         return [r._asdict() for r in results]
+
+    def get_business_info_by_abn(self, email_address, abn):
+        apiKey = current_app.config['ABR_API_KEY']
+        includeHistoricalDetails = 'N'
```

Comment:
Should we stick with the convention of lower case variable names (eg. `api_key` and `include_historical_details`)?

Reply:
resolved